databases/week1/week1_HW.sql

```diff
@@ -0,0 +1,68 @@
+use hyf_class2023;
+
+-- 1 Find out how many tasks are in the task table
+
+SELECT count(id)
+FROM task;
+
+-- 2 Find out how many tasks in the task table do not have a valid due date
+
+SELECT count(id)
+FROM task
+where due_date is null;
+
+-- 3 Find all the tasks that are marked as done
+
+select title, status_id
+from task
+where status_id='3';
+
```

Comment:
How do I / some one else know status_id= 3 means Done, 

```
SELECT title,status.name 
FROM task
JOIN status ON task.status_id=status.id
WHERE status.name="Done";
```

Reply:
Hi Elia, Thank you so much for 

> 

Hi Elia. Thank you so much for reviewing my HW. I did the empty lines for readability but i am removing them.